usaspending_api/transactions/delta_models/published_fabs.py

```diff
@@ -0,0 +1,120 @@
+PUBLISHED_FABS_COLUMNS = {
+    "published_fabs_id": {"delta": "INTEGER", "postgres": "UUID"},
```

Comment:
Any reason you're converting this to a `UUID` here?

Reply:
addressed